index.html

```diff
@@ -667,6 +667,52 @@ <h4>Use the new JSON-LD context when issuing new credentials</h4>
 
       </section>
 
+      <section>
+        <h3>Extending JWTs</h3>
+
+        <p>
+The VC Data Model specifies a minimal set of JWT <a>claim</a> names that must be used
+to represent <a>verifiable credential</a> properties and <code>credentialSubject</code>
+properties. Implementers may wish to extend a <a>verifiable credential<a> with a
+property that is either new e.g. drivingLicenseNumber, or that is already defined 
+as a JWT claim with IANA e.g.<a>given_name</a>. It is recommended that these properties
+are placed directly in either the JWT <code>vc</code> <a>claim</a> or
+<code>credentialSubject</code> property of the <code>vc</code> <a>claim<a> as appropriate.
+        </p>
+        <p>
+However, if implementers wish to use JWT <a>claim<a> names for these extensions, 
+the following steps are recommended. Note that there are three types of JWT
+<a>claim<a> name: public, named with a URI; private, named with a local name; and 
+registered with IANA.
+        <ol>
+          <li>
+First, check with IANA (https://www.iana.org/assignments/jwt/jwt.xhtml) to see
+if the JWT <a>claim<a> name already exists.
+          </li>
+          <li>
+If it does not exist, the implementer may wish to either give it a public name 
+(i.e., a URI), or give it a local name (i.e., any string), or register it with IANA.
```

Comment:
```
either give it a public name 
(i.e., a URI), or give it a local name (i.e., any string), or register it with IANA.
```
->
```
either give it a public name 
(i.e., a URI), give it a local name (i.e., any string), or register it with IANA.
```

that is, delete the first "or"

Reply:
This extra "or" still remains.  `or give` should be just `give".

Reply:
Missed that one, thanks